tools/wpt/browser.py

```diff
@@ -636,6 +636,9 @@ def version(self, binary=None, webdriver_binary=None):
         # "Included with Safari 12.1 (14607.1.11)"
         # "Included with Safari Technology Preview (Release 67, 13607.1.9.0.1)"
         # The `--version` flag was added in STP 67, so allow the call to fail.
+        # For the two examples, this function will return, respectively:
+        # "12.1 (14607.1.11)"
+        # "67 (13607.1.9.0.1)"
```

Comment:
Why delete the channel information? We keep it for Chrome, and it's represent as an "a" in the Firefox version.

Reply:
Yeah we can have the channel at the end. What about "67 (13607.1.9.0.1) Preview"? (Or perhaps the full "Technology Preview"?)

Reply:
What is the code that will end up parsing this string for wpt.fyi? I feel like this is to avoid changing that but don't understand why. It seems very likely there will be occasional changes to the version string formats that existing code handles incorrectly, and we'll have to fix it after the fact. Doing too much parsing here might mangle it in the interim.

Reply:
https://github.com/web-platform-tests/wpt/issues/14465#issuecomment-446773143